test/fixtures/sub-process.js

```diff
@@ -4,7 +4,15 @@ var Insight = require('../../lib');
 var insight = new Insight({
 	packageName: 'yeoman',
 	packageVersion: '0.0.0',
-	trackingCode: 'GA-1234567-1'
+	trackingCode: 'GA-1234567-1',
+	config: {
```

Comment:
We should just rely on sinon to mock this object and make assertion against the call. Reading from the stderr/out for assertion purposes is kind of dirty.